tests/ha/test_multi_server_pod_failure.py

```diff
@@ -629,3 +629,383 @@ def test_writes_till_kserver_pods_fail(self):
 
         LOGGER.info("Completed: Test to verify degraded WRITEs after each pod failure till K "
                     "server pods down - unsafe shutdown.")
+
+    # pylint: disable=too-many-statements
+    @pytest.mark.ha
+    @pytest.mark.lc
+    @pytest.mark.tags("TEST-40579")
+    @CTFailOn(error_handler)
+    def test_writes_kserver_pods_failure(self):
+        """
+        Test to verify degraded WRITEs after all K server pods down - unsafe shutdown.
+        """
+        LOGGER.info("Started: Test to verify degraded WRITEs after all K server pods "
+                    "down - unsafe shutdown.")
+
+        LOGGER.info("STEP 1: Perform WRITEs-READs-Verify with variable object "
+                    "sizes. 0B + (1KB - 512MB)")
+        users = self.mgnt_ops.create_account_users(nusers=1)
+        self.test_prefix = 'test-40579'
+        self.s3_clean = users
+        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                    log_prefix=self.test_prefix,
+                                                    skipcleanup=True, nsamples=2, nclients=2)
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Step 1: Performed WRITEs-READs-Verify with variable sizes objects.")
+
+        LOGGER.info("Step 2: Shutdown the server pod by deleting deployment (unsafe)")
+        LOGGER.info("Get server pod names to be deleted")
+        pod_list = self.node_master_list[0].get_all_pods(pod_prefix=const.SERVER_POD_NAME_PREFIX)
+        server_data_pod_list = pod_list.extend(self.node_master_list[0].get_all_pods
```

Comment:
extend don't return anything.
it will just update the pod_list. SO either you extend the server_data_pod_list or define new list having both the type of pods